weave-js/src/components/PagePanelComponents/Home/Browse3/feedback/Notes.tsx

```diff
@@ -39,8 +39,8 @@ export const Notes = ({
       }
     : undefined;
   const onKeyDown = onNoteAdded
-    ? (key: string) => {
-        if (key === 'Enter') {
+    ? (key: string, e: React.KeyboardEvent<HTMLInputElement>) => {
+        if (key === 'Enter' && !e.nativeEvent.isComposing) {
```

Comment:
Wow, had no idea that was a thing, nice!

Reply:
Neither did I before yesterday!